test/fixtures/nodes/node01.json

```diff
@@ -1,27 +1,23 @@
 {
-  "name": "node01",
-  "chef_environment": "_default",
-  "json_class": "Chef::Node",
-  "automatic": {
-    "hostname": "vagrant.vm",
-    "recipes": [
-      "elasticsearch::default"
-    ],
-    "roles": [
-    ],
-    "ipaddress": "192.168.0.1"
-  },
-  "tags": [
-    "app"
-  ],
-  "normal": {
-  },
-  "chef_type": "node",
-  "default": {
-  },
-  "override": {
-  },
-  "run_list": [
-    "recipe[elasticsearch::default]"
-  ]
+	"name": "node01",
+	"chef_environment": "_default",
+	"json_class": "Chef::Node",
+	"automatic": {
+		"hostname": "vagrant.vm",
+		"recipes": [
+			"elasticsearch::default"
+		],
+		"roles": [],
+		"ipaddress": "192.168.0.1"
+	},
+	"tags": [
+		"app"
+	],
+	"normal": {},
+	"chef_type": "node",
+	"default": {},
+	"override": {},
+	"run_list": [
+		"recipe[elasticsearch::default]"
```

Comment:
Should this be removed if the intent is to remove the default recipe?

Reply:
I just left it in here for now. This is very much a work in progress.